packaging/generic-rpm/ecs-init.spec

```diff
@@ -0,0 +1,99 @@
+# Copyright 2014-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+%global gobuild_tag al2
```

Comment:
Should this tag still be al2?

Reply:
i suppose we should change the tag, and we should fix the existing build tag divergence. Since this is needed for the deb package too maybe we can follow up with a separate PR

Reply:
unfortunately if the tag is unspecified, it defaults to what al1 wants and uses a cgroup location that only works for al1 https://github.com/aws/amazon-ecs-init/blob/master/ecs-init/config/config_unspecified.go#L19. so if we want to not specify build tag here, we need to add a build tag for al1 and make sure that works. i added a comment here about it and we can follow up later